doc/basics.md

```diff
@@ -53,6 +53,14 @@ rustup-toolchain-install-master -f -n master -c rustc-dev -c llvm-tools
 rustup override set master
 ```
 
+**Note:** You may get compiler errors when building Clippy, even if you
+didn't change anything. That's because, as mentioned earlier, Clippy
+depends on rustc internals. There is [discussion][nightly-build-issue]
+on ways to make sure that rustc does not break Clippy in the future
+so that Clippy always builds.
+
+[nightly-build-issue]: https://github.com/rust-lang/rust-clippy/issues/5561
+
```

Comment:
```suggestion
_Note:_ Sometimes you may get compiler errors when building Clippy, even if you
didn't change anything. Normally those will be fixed by a maintainer in a few hours. 

```

and put it in front of the code block, so you don't need to mention the internals again.

Also the issue is not relevant for beginners.